mod-audit-server/src/main/java/org/folio/dao/acquisition/OrderEventsDao.java

```diff
@@ -15,4 +18,12 @@ public interface OrderEventsDao {
    * @return future with created row
    */
   Future<RowSet<Row>> save(OrderAuditEvent orderAuditEvent, String tenantId);
+
+  /**
+   * Searches for OrderAuditEvent by id
+   *
+   * @param id OrderAuditEvent id
+   * @return future with optional OrderAuditEvent
+   */
+  Future<OrderAuditEventCollection> getAcquisitionOrderAuditEventById(String id, int limit, int offset, String tenantId);
```

Comment:
We need to rename this method like this:
getAuditEventsByOrderId(String orderId, int offset, int limit, String tenantId);

Also need to rename java doc because we can return multiple events:
Searches for order audit events by order id

Reply:
fixed